public/sass/components/_gf-form.scss

```diff
@@ -139,8 +147,15 @@ $gf-form-label-margin: 0.25rem;
   }
 }
 
-.gf-form-select-wrapper + .gf-form-select-wrapper { 
-  margin-left: $gf-form-label-margin; 
+.gf-form--textarea {
+  align-items: flex-start;
+}
+
+input[type="number"].gf-natural-language-form {
```

Comment:
not sure I understand this one? the naming is a bit strange, pixel margins, should this not be on gf-form-input (with &in put[type=number] specifier ?